corecito.py

```diff
@@ -54,19 +62,38 @@ async def main():
 
       elif coreNumberIncreased(account.core_number, deviated_core_number, account.min_core_number_increase_percentage, account.max_core_number_increase_percentage):
         logger.logCoreNumberIncreased(increase_percentage, excess, account.core_number_currency, account.base_currency, telegram)
-        tx_result = round(excess * buy_price, account.max_decimals_buy)
+        #Check if 'fiat' is True to adjust messages format and tx_result var and 'excess' has to be divided by the buy_price
+        if fiat:
+          tx_result = round(excess / buy_price, account.max_decimals_buy)
+        else:
+          tx_result = round(excess * sell_price, account.max_decimals_buy)
         logger.logSellExcess(tx_result, account.base_currency, buy_price, excess, account.core_number_currency, telegram)
```

Comment:
I just realize that the logged price here(buy_price) and in `coreNumberDecreased` are incorrect for fiat, because they use the other side of the order book. Maybe it's better to extract the price for the transaction and the log to a `tx_price` variable.
What do you think?

Reply:
yeah, it makes sense, nice catch, I will update it right now.

Reply:
Done!